backend/fms_core/migrations/0019_v3_1_2.py

```diff
@@ -0,0 +1,76 @@
+from django.conf import settings
+from django.db import migrations
+from django.contrib.auth.models import User
+import reversion
+import datetime
+
```

Comment:
I would add at the top a comment that explains the issue that is fixed by this data migration. The migration if applied to another DB running the same app would be useless. We may also like to get a clearer idea of what is happening here 2 years from now when we can to do some migration cleanup. Just a short comment explaining that between version 3.1 and 3.1.1 (which correspond the dates below) a bug was generating a process for each line of the submitted templates for extraction, transfer and update.